api/src/main/java/org/pmiops/workbench/api/CohortController.java

```diff
@@ -85,7 +87,17 @@ public Cohort apply(org.pmiops.workbench.db.model.Cohort cohort) {
     dbCohort.setWorkspaceId(workspace.getWorkspaceId());
     dbCohort.setCreationTime(now);
     dbCohort.setLastModifiedTime(now);
-    dbCohort = cohortDao.save(dbCohort);
+    try {
```

Comment:
@danqrodney What do you think?
* Catching an SQL integrity exception doesn't seem ideal, but I'm not sure it's worth adding a DAO method to look up by external ID + wrapping the lookup and insert in a transaction.
* I'd really like to report the error message to the client, or failing that at least log it on the server. I don't see a way to override the "reason" string on exceptions. Maybe we should switch to custom exception handlers?